kubeflow/core/configure_envoy_for_iap.sh

```diff
@@ -9,7 +9,6 @@
 [ -z ${NAMESPACE} ] && echo Error NAMESPACE must be set && exit 1
 [ -z ${SERVICE} ] && echo Error SERVICE must be set && exit 1
 
-apk add --update jq
 curl https://storage.googleapis.com/kubernetes-release/release/v1.9.4/bin/linux/amd64/kubectl > /usr/local/bin/kubectl && chmod +x /usr/local/bin/kubectl
```

Comment:
Add this curl to the Dockerfile and delete from this script. Also bump it to v1.11.0.